documentation/slate/source/includes/integration.md

```diff
@@ -806,10 +781,10 @@ access it via that kind of url: <https://api.navitia.io/v1/{a_path_to_a_resource
 
 Required | Name             | Type      | Description                                                                              | Default Value
 ---------|------------------|-----------|------------------------------------------------------------------------------------------|--------------
-yep      | from_datetime    | date_time | The date_time from which you want the schedules                                          |
+yep      | from_datetime    | [iso-date-time](#iso-date-time) | The date_time from which you want the schedules                                          |
 nop      | duration         | int       | Maximum duration in seconds between from_datetime and the retrieved datetimes.           | 86400
 nop      | max_date_times   | int       | Maximum number of columns per schedule.                                                  |
-nop      | data_freshness   | enum      | Define the freshness of data to use<br><ul><li>real_time</li><li>base_schedule</li></ul> | base_schedule
+nop      | data_freshness   | enum      | Define the freshness of data to use<br><ul><li>realtime</li><li>base_schedule</li></ul> | base_schedule
```

Comment:
I think you missed the `forbidden_uris[]` parameter here (https://github.com/CanalTP/navitia/commit/bae95b4ce6f46cc9c46bd4c8e014771fb1b684d5)